lib/elixir/lib/registry.ex

```diff
@@ -1369,7 +1369,7 @@ defmodule Registry.Partition do
   The process is only responsible for monitoring, demonitoring
   and cleaning up when monitored processes crash.
   """
-  def start_link(registry, arg) do
+  def start_link([registry, arg]) do
```

Comment:
If you merge the above, then this should be rolled back:

```suggestion
  def start_link(registry, arg) do
```